webpack.config.js

```diff
@@ -138,6 +138,7 @@ const config = {
     }),
     new webpack.NamedModulesPlugin(),
     new webpack.ProvidePlugin({
+      "window.$": "jquery",
```

Comment:
Force is old and spread out... a few more areas that might be relevant! 

https://github.com/damassi/force/blob/3ae4927a42a780e9c2c66073c78de5ab2250d498/src/lib/global_modules.js#L1

https://github.com/damassi/force/blob/3ae4927a42a780e9c2c66073c78de5ab2250d498/src/desktop/lib/global_client_setup.coffee#L7

(The latter has a similar setup under `src/mobile/lib`)

Reply:
Thanks for the links! I'm not having much luck narrowing down the issue but would love to pair on it tomorrow if I don't figure it out by then.